project_generation/COMPLETE_PROJECT_SETUP.md

```diff
@@ -53,17 +54,21 @@ $ dp generate-project --create-repository yes --name {name-of-repository} --proj
 
 For example `dp-topic-api` was created with:
 
-`
-$ dp generate-project --create-repository yes --description "Enables greater flexibility in creating journeys through the website" --go-version 1.13 --name dp-topic-api --port 25300 --project-location . --strategy git --type api
-`
+```shell
+$ dp generate-project --create-repository yes --description "Enables greater flexibility in creating journeys through the website" --go-version 1.17 --name dp-topic-api --port 25300 --project-location . --strategy git --type api
```

Comment:
Would it be worth using a generic name (e.g. dp-helloworld-api) in this command; the dp-topic-api is a real api

Reply:
If you agree, you probably want to update the markdown by replacing all references of `dp-topic-api` to `dp-helloworld-api`

Reply:
I don't agree, it's used as an example - see above

Reply:
You are right, I did not see line 55